### PR TITLE
Single input commenting/javokhir

### LINKED ISSUE #160: Multiple Comment Input Boxes Simultaneously
**Is your feature request related to a problem? Please describe.**
It's unnecessary to have multiple text boxes open when commenting discussing. I think our UI could benefit from improving the flow of commenting de novo, replying, and editing a comment. 

**Describe the solution you'd like**
If I comment, reply, to a comment, and edit, there should never be anything but a single input text box

**Describe alternatives you've considered**
I think we should find a platform -- Reddit, Youtube, Discord, etc -- and mimic them. I want to get the best UI possible. 

**Additional context**
See attached
localhost_9200_2022_12_i-won-t-edit-you-but-i-will-give-you-an-entry(iPhone SE)

### PR DESCRIPTION
**Pull request by NJavokhir**
Resolve #160 

One single input is used for creating new comment and replying comment. Leave your comment...

### CODE REVIEW

File: public/icons/arrow-reply-svgrepo-com.svg
Comment (arnonrdp) [inline public/icons/arrow-reply-svgrepo-com.svg:1]: Please rename it and remove `-svgrepo-com` from the name.

File: src/components/TheComments.vue [hunk 1]
```diff
@@ -24,20 +27,35 @@
             rounded
           >
             <q-list>
-              <q-item data-test="comment-select-edit" clickable v-close-popup @click="editInput(comment.id)">
+              <q-item
+                data-test="comment-select-edit"
+                clickable
+                v-close-popup
+                @click="editInput(comment.id)"
+              >
```
Comment (arnonrdp) [inline src/components/TheComments.vue:35]: Most changed lines do not have an actual change.
It's just your IDE applying different formatting. Please follow `.prettierrc`, use `Volar` and remove `Vetur`. If you need help with these settings let me know.

File: test/cypress/e2e/comment.cy.js [hunk 2]
```diff
@@ -11,14 +11,15 @@ entryValues.forEach((entry) => {
...
-          cy.visit('/2022/11/applications')
+        if (cy.visit('/2022/11/applications').wait(5000)) {
+          cy.visit('/2022/11/applications').wait(5000)
         } else {
-          cy.visit('/2022/11/-diane-m-m-daryl-mccormack-unshaken-we-ve-heard')
+          cy.visit('/2022/11/-diane-m-m-daryl-mccormack-unshaken-we-ve-heard').wait(5000)
         }
+      } else {
+        cy.visit('/month').wait(3000)
       }
```
Comment (arnonrdp) [inline test/cypress/e2e/comment.cy.js:23]: It's all very static here. Can't we just use `/month`?

**Review by arnonrdp — CHANGES_REQUESTED** [review]
Reply functionality is still not working as expected.

Note that:
- When adding a reply, it is added to the two comments;
- When reloading the page, one of the comments disappeared;
- There was no sign of loading on the page;

Also take a look at removing comments. I had a problem with this later but it was no longer recording the screen.

File: test/cypress/e2e/comment.cy.js (hunk 2 above)
Comment (NJavokhir) [reply test/cypress/e2e/comment.cy.js:23]: yeah, you are right. we can test only '`/month`'

**Review by arnonrdp — APPROVED** [review]

**Pull request merged**